recipes/default.rb

```diff
@@ -22,6 +22,28 @@
 unless Homebrew.exist?
   homebrew_go = "#{Chef::Config[:file_cache_path]}/homebrew_go"
 
+  # Grant Homebrew install script permission to execute
+  # without passing a sudo password. Deletes itself at
+  # the end of a successful run.
+  sudo 'nopasswd_homebrew_installer' do
```

Comment:
This will mean every time we run this it will have a change on disk, which feels wrong, is there a way to do this in a more idempotent way?

Reply:
I could add a not_if guard to the resource for extra protection, but since this is already wrapped in the `unless Homebrew.exist?` check, the `sudo` block should only execute on the first run without homebrew installed.

Reply:
Seems like a decent way of doing it to me 👍🏼